source/reference/mongosync.txt

```diff
@@ -150,6 +150,24 @@ may differ between source and destination.
 Transactions may appear non-atomically on the destination cluster.
 Retryable writes may not be retryable on the destination cluster. 
 
+Profiling
+~~~~~~~~~
+
+If profiling is enabled on a source database, MongoDB creates a special
+collection, ``<db>.system.profile``. After syncing,
+{+c2c-product-name+} will not drop the ``<db>.system.profile``
+collection from the destination even if the source database is dropped
+at a later time. The empty ``<db>.system.profile`` collection will not
```

Comment:
Sorry for the drive-by comment, unlike `system.views`, `system.profile` is not necessarily empty. The slow query entries recorded because of the profiling would still remain in `<db>.system.profile` collection.